source/sass/views/home.scss

```diff
@@ -1,5 +1,40 @@
-.cta {
-  min-height: 350px;
+#view-home {
+  $section-padding-y: 5rem;
+
+  .section-quote {
```

Comment:
Question out of curiosity, and feel free to ignore this if you like — would the final .css file be smaller if there was less nested scss? It might not make enough of a difference here to bother with to be honest but thought I'd bring it up.

Reply:
Good point! Usually, I don't nest SCSS this much or target a very specific selector. However, since the design of this homepage quote section is unique and doesn't follow our brand guide, I'm making the selectors very specific (nested SCSS) so the rules don't affect the rest of the site.